Lib/blackrenderer/backends/coregraphics.py

```diff
@@ -207,17 +208,18 @@ def drawPathSweepGradient(
     # TODO: blendMode for PaintComposite
 
 
-def _unpackColorLine(colorLine):
+def _unpackColorLine(colorLine, colorSpace):
     colors = []
     stops = []
     for stop, color in colorLine:
-        colors.append(CG.CGColorCreateGenericRGB(*color))
+        colors.append(CG.CGColorCreate(colorSpace, color))
         stops.append(stop)
     return colors, stops
 
 
 class CoreGraphicsPixelSurface(Surface):
     fileExtension = ".png"
+    colorSpace = CG.CGColorSpaceCreateWithName(CG.kCGColorSpaceSRGB)
```

Comment:
I'm doubting whether this should perhaps be an argument to `__init__` instead, and become an instance attr.

Do you know whether creating a CGColorSpace object is expensive?

Reply:
Thanks. `CGColorSpaceCreateWithName(kCGColorSpaceSRGB)` seems to return an interned instance and it takes only < 100ns per call on my Mac. Not that slow, but at the same time I think there's no particular reason not to reuse the instance.

Reply:
Would it make sense to use a global constant for the default color space instance?

Reply:
I agree that makes more sense in Python. Thanks!